server/src/main/resources/openapi/shared-index-1.0.yaml

```diff
@@ -334,6 +334,16 @@ paths:
           $ref: "#/components/responses/trait_400"
         "500":
           $ref: "#/components/responses/trait_500"
+    delete:
```

Comment:
A general comment not specific to this PR: should some of the params be moved to lower level if they don't apply to all methods?

Reply:
Possibly. The delete, however, uses query. I'll see if can mix it.